library/std/src/sys/unix/fs.rs

```diff
@@ -925,7 +925,7 @@ impl File {
             SeekFrom::End(off) => (libc::SEEK_END, off),
             SeekFrom::Current(off) => (libc::SEEK_CUR, off),
         };
-        let n = cvt(unsafe { lseek64(self.as_raw_fd(), pos, whence) })?;
+        let n = cvt(unsafe { lseek64(self.as_raw_fd(), pos as libc::off_t, whence) })?;
```

Comment:
Perhaps it would be better to use `try_into` here, and fail if we can't fit the seek request into `libc::off_t`

Reply:
I tried this but doesn't work because there is not a `try_into` for `i64` to `i64`